docs/0.1-Glossary.md

```diff
@@ -0,0 +1,44 @@
+
+# 0.1 Glossary
+This glosary is intended to catalogue and define all terms used in our documentation which may not be evident to new users of the operator.
+
+By extension of this glossary, we may also use terms from the following glossaries:
+* [OLM](https://docs.openshift.com/container-platform/4.7/operators/understanding/olm-common-terms.html)
+* [Kubernetes](https://kubernetes.io/docs/reference/glossary/?fundamental=true)
+
+
+## Terms
+### Payer Account
+[AWS Organizations](https://aws.amazon.com/organizations/) give you a "unified bill" so to speak under a single account. Sub accounts are created under this "Payer Account", which is the highest-level account in this organization. SREs all should also have an IAM user with credentials on each payer account, which has AssumeRole Permissions as administrator into each of these sub-accounts.
+
+### Two Cluster Types
+We support 2 types of clusters:
+- **Red Hat Managed Cluster:** Standard OpenShift Dedicated clusters are deployed into their own cloud infrastructure accounts, each owned by Red Hat. Red Hat is responsible for this account. From this account, only Red Hat managed clusters will be visible, in other words, CCS/BYOC clusters will not.
+- **CCS / BYOC:** CCS *(Customer Cloud Subscriptions)* or BYOC *(Bring your Own Cluster)* clusters are those which belong to customers’ personal AWS accounts, as opposed to being owned by Red Hat. Therefore, for CCS we are provisioning a cluster in an account provided by the customer. For CCS clusters, we are going to find multiple clusters per account.
+
+### STS
+[AWS Security Token Service (AWS STS)](https://docs.aws.amazon.com/STS/latest/APIReference/welcome.html) is a web service that enables you to request temporary, limited-privilege credentials for AWS Identity and Access Management (IAM) users or for users that you authenticate (federated users).
+
+### Trust Relationship
+Trust relationships/Trust Policies are then configured between the IAM users and the IAM roles. This policy defines which principals can assume the role, and under which conditions. This is sometimes referred to as a resource-based policy for the IAM role.
+
+### Assume Role
+Returns a set of temporary security credentials that you can use to access AWS resources that you might not normally have access to. These temporary credentials consist of an access key ID, a secret access key, and a security token. Typically, you use AssumeRole within your account or for cross-account access.
```

Comment:
We could link the respective AWS doc here. This [1] or something similar.

[1] https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRole.html